openc3-cosmos-init/plugins/packages/openc3-js-common/src/utils/formatter.js

```diff
@@ -0,0 +1,29 @@
+/*
+# Copyright 2025, OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+*/
+
+const formatBytesToString = function(bytes) {
+  if (bytes === 0) return '0 B'
+  
+  const sizes = ['B', 'KB', 'MB', 'GB', 'TB']
+  const k = 1024
```

Comment:
The International Electrotechnical Commission recommends using 1000 for decimal prefixes KB, MB, GB and 1024 for binary prefixes KiB, MiB, GiB. When I download a file that is 2,405,569 bytes, Mac OS says it is 2.4 MB on disk. Your calculation is 2.29 MB. Let's change this to 1000 to match the OS disk calculation.